### ISSUE TITLE
Required version of helhum/typo3-console not checked before deployment

### ISSUE THREAD

**Issue by YetiCGN** [TYPO3/Surf#175]
## Expected Behavior

When `composer require`ing Surf it should a) require at least `"helhum/typo3-console": "^5.4"` or b) suggest it for TYPO3 CMS projects but then define a conflicting version with typo3-console < 5.4.0.

## Actual Behavior

Surf checks whether the console is available, but it doesn't check the version so packages of `helhum/typo3-console` < 5.4.0 will throw the following error during deployment:
```
live (TYPO3 CMS) TYPO3\Surf\Task\TYPO3\CMS\SetUpExtensionsTask
> [ Symfony\Component\Console\Exception\CommandNotFoundException ]
> The command "extension:setupactive" does not exist.
```

## Steps to Reproduce the Problem

  1. Require both `"helhum/typo3-console": "^5.2"` and `"typo3/surf": "dev-master"` in composer.json
  1. Run any TYPO3 CMS deployment task within that project

## Specifications

  - Surf Version: dev-master @ 4dfae0b
  - Application: TYPO3 CMS 8.7.17
  - PHP Version: 7.1.18
  - Platform: Debian GNU/Linux 9.5 (stretch)
  - Environment (CI): GitLab CI
  - Deployment configuration: ?

**Comment by YetiCGN** [comment]
Never mind, I was mistaken. `extension:setupactive` is already available in 5.2.0, just not showing up in my setup for some reason. Need to dig deeper.

**Comment by t3easy** [comment]
Surf doesn't require typo3-console... E.G. if you deploy Neos.
